extension/src/experiments/index.ts

```diff
@@ -353,8 +349,10 @@ export class Experiments extends BaseRepository<TableData> {
     return this.experiments.getWorkspaceCommitsAndExperiments()
   }
 
-  public async selectExperiments() {
-    const experiments = this.experiments.getWorkspaceCommitsAndExperiments()
+  public async selectExperimentsToPlot() {
```

Comment:
[F] This is the only thing this function is used for so I think the name better explains the usage